testdata/p4_16_samples/bitvec_hdr.p4

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
You need to check-in the outputs generated by the compiler when running on this test. You will see them once you run make check; they will appear as new files in git status.

Reply:
I see the files in git status - thanks.  I will check them in a next commit.  I have also fixed the failure in negative tests header2_e.p4 and issue407-1.p4.  The failures were due to the P4 header now allowing a struct.

Reply:
Click on "details" and you will see the problem. A reference output has changed.

Reply:
You either did not run in the correct branch, or if you added the test you forgot to run 'cmake ..'